python/docs/conf.py

```diff
@@ -47,17 +47,17 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'GraphFrames'
-copyright = u''
+project = u'pygraphframes'
+copyright = u'2016, Author'
```

Comment:
But also Ankur, Feynman, Xiangrui, and hopefully others in the future.  I'd prefer to leave it generic.